commit e875d33d7f06d1107c057d12bb5aaba84738e418
Author: Tobin C. Harding <me@tobin.cc>
Date:   Mon Feb 26 09:27:59 2018 +1100

    MAINTAINERS: Update LEAKING_ADDRESSES
    
    MAINTAINERS is out of date for leaking_addresses.pl. There is now a tree on
    kernel.org for development of this script.  We have a second maintainer now,
    thanks Tycho.  Development of this scripts was started on kernel-hardening
    mailing list so let's keep it there.
    
    Update maintainer details; Add mailing list, kernel.org hosted tree, and second
    maintainer.
    
    Signed-off-by: Tobin C. Harding <me@tobin.cc>

diff --git a/MAINTAINERS b/MAINTAINERS
index 6e950b8b4a41..fef8f4ad7c3e 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7910,7 +7910,10 @@ F:	drivers/scsi/53c700*
 
 LEAKING_ADDRESSES
 M:	Tobin C. Harding <me@tobin.cc>
+M:	Tycho Andersen <tycho@tycho.ws>
+L:	kernel-hardening@lists.openwall.com
 S:	Maintained
+T:	git git://git.kernel.org/pub/scm/linux/kernel/git/tobin/leaks.git
 F:	scripts/leaking_addresses.pl
 
 LED SUBSYSTEM